week-1/Homework/extra/2-piping.js

```diff
@@ -16,43 +16,49 @@
   the final result to the variable goodCode
 */
 
-function add() {
-
+function add(num1, num2) {
+  //return parseFloat(num1) + parseFloat(num2).toFixed(1); //test failed
+  return (Number(num1) + Number(num2)).toFixed(1); //test failed
+  //return Number(num1) + Number(num2);
 }
 
-function multiply() {
-
+function multiply(num1, num2) {
+  return parseFloat(num1 * num2);
 }
 
-function format() {
-
+function format(price) {
+  return `£${price}`;
 }
 
-const startingValue = 2
+const startingValue = 2;
 
 // Why can this code be seen as bad practice? Comment your answer.
-let badCode = 
+//I repeated myself and that's why this is bad code
+badCode = `£${(startingValue + 10) * 2}`;
```

Comment:
How could you use your functions to output the exact same results as above? Remember, you already wrote functions that perform these individual operations. 🔢

Reply:
I did not use the function because that is a bad code , I use the function in good code variable `let goodCode = format((startingValue + 10) * 2);`